File: app/src/main/java/fr/paug/androidmakers/util/EmojiUtils.java
package fr.paug.androidmakers.util;

public final class EmojiUtils {

    public static String getLanguageInEmoji(String language) {
        if ("en".equalsIgnoreCase(language)) {
            return "\uD83C\uDDEC\uD83C\uDDE7";
        } else if ("fr".equalsIgnoreCase(language)) {
            return "\uD83C\uDDEB\uD83C\uDDF7";
        } else {
            return "\uD83C\uDDEC\uD83C\uDDE7 \uD83C\uDDEB\uD83C\uDDF7";
        }
    }

}

File: app/src/main/java/fr/paug/androidmakers/util/EmojiUtils.kt
package fr.paug.androidmakers.util

import java.util.*

object EmojiUtils {

    fun getLanguageInEmoji(language: String?): String? {
        return when (language?.toLowerCase(Locale.ROOT)) {
            "english" -> "\uD83C\uDDEC\uD83C\uDDE7"
            "french" -> "\uD83C\uDDEB\uD83C\uDDF7"
            else -> null
        }
    }
}